backend/src/main/java/capstone/storage/backend/exceptions/ExceptionMessage.java

```diff
@@ -0,0 +1,26 @@
+package capstone.storage.backend.exceptions;
+
+public enum ExceptionMessage {
```

Comment:
Lagere diese Texte in die jeweiligen Exception-Klassen aus.
Du siehst ein Beispiel in meinem Kommentar zu `IsNullOrEmptyException`.